package.json

```
@@ -67,7 +67,7 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-config-standard": "^14.1.1",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsdoc": "^24.0.6",
+    "eslint-plugin-jsdoc": "^25.0.0",
```

We can simplify this to:
```suggestion
    "eslint-plugin-jsdoc": "25",
```